README.md

```diff
@@ -98,7 +98,7 @@ Major components of the project include:
 
 ## Documentation
 
-The documentation is available at <https://opendal.apache.org>.
+The documentation is available at <https://opendal.apache.org>. Checkout [here](./examples/) for the usage examples(still working on).
```

Comment:
```suggestion
The documentation is available at <https://opendal.apache.org>. Checkout [here](./examples/) for the usage examples(still being worked on).
```

Reply:
Let's send a new PR for this instead adding new workload for contributor.